scripts/leapfrog/pre_redeploy.sh

```diff
@@ -74,6 +88,14 @@ pushd ${LEAPFROG_DIR}
             fi
         popd
         log "variable-migration" "ok"
+        # This message only needs to appears the first time the user variable migration was successful
+        # This way, if a user mistakenly Ctrl-C, the variable migration process will be skipped on
+        # the next run
+        if [[ "${IS_GATING}" != "TRUE" ]]; then
+            notice "Please verify your migrated secrets in /etc/openstack_deploy"
+            warning "DO NOT CTRL-C this process to verify your secrets."
```

Comment:
If someone does it, it will imply a need for workarounding, or weird expectations, as expressed before.

It's better if we do it that way, IMO, for user friendliness.